Homeworks/homework_16/base_suite.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library           SeleniumLibrary
+Variables         variabels.py
+Resource          common_keywords.resource
+
+*** Test Cases ***
+User Registration and Login Test
+    Open Browser and Log In
+    [Teardown]    Close Browser
+
+Product Category Test
+    Open Browser and Log In
```

Comment:
`[Setup]  Open Browser and Log In`
would be better because login is considered as a precondition for this case